shiksha-migration/cohort-members-userdb-migration.js

```
@@ -0,0 +1,196 @@
+        //  * CREATE MEMBERSHIP MAPPING VIA API
+        //  */
+        await axios.post(
+          `${apiBaseUrl}/user/v1/cohortmember/create`,
+          {
+            userId: member.userId,
+            cohortId: member.cohortId,
+            cohortAcademicYearId: member.cohortAcademicYearId,
+          },
+          {
+            headers: {
+              'accept': '*/*',
```

ESLint

[error] Replace `'accept'` with `accept`

(prettier/prettier)